source/views/dictionary/list.js

```diff
@@ -77,21 +99,43 @@ export class DictionaryView extends React.Component {
     return <ListSeparator key={`${sectionId}-${rowId}`} />
   }
 
+  handleResults = (results: Object) => {
+    this.setState({results: results})
+  }
+
   render() {
+    if (!terms) {
+      return <LoadingView />
+    }
+
     return (
-      <StyledAlphabetListView
-        data={groupBy(terms, item => head(item.word))}
-        cell={this.renderRow}
-        // just setting cellHeight sends the wrong values on iOS.
-        cellHeight={
-          rowHeight +
-            (Platform.OS === 'ios' ? 11 / 12 * StyleSheet.hairlineWidth : 0)
-        }
-        sectionHeader={this.renderHeader}
-        sectionHeaderHeight={headerHeight}
-        showsVerticalScrollIndicator={false}
-        renderSeparator={this.renderSeparator}
-      />
+      <View style={styles.wrapper}>
+        <SearchBar
+          data={terms}
+          handleResults={(results) => this.handleResults(results)}
+          closeButton={<Icon style={styles.closeIcon} name="ios-close" />}
+          showOnLoad={true}
+          hideBack={true}
+          allDataOnEmptySearch={true}
+          autoCorrect={false}
+          focusOnLayout={true}
+          iOSPadding={false}
```

Comment:
`iOSPadding` is set to true by default. The size of the searchbar would be equivalent to the android size in the screenshots. I was under the impression it should be smaller. Do you have any thoughts about that?